developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Update Aptos Validator Node"
+slug: "update-validator-node"
+---
+
+# Update Aptos Validator Node via Failover
```

Comment:
Should we also mention that unless the code is modified, running backup nodes doesn't have slashing risks? This is different from running a validator node for some other networks.

Reply:
I don't think we should mention this. I suspect we'll actually want to do some form of slashing/rejections if someone runs a backup validator in the future :/